src/base/bittorrent/session.cpp

```diff
@@ -1982,7 +1983,7 @@ void Session::increaseTorrentsPriority(const QStringList &hashes)
             std::greater<QPair<int, TorrentHandle *> > > torrentQueue;
 
     // Sort torrents by priority
-    foreach (const InfoHash &hash, hashes) {
+    for (const InfoHash hash : hashes) {
```

Comment:
why remove the `&`?

Reply:
warning: loop variable 'hash' has type 'const BitTorrent::InfoHash &' but is initialized with type 'const QString' resulting in a copy
use non-reference type 'BitTorrent::InfoHash' to keep the copy or type 'const QString &' to prevent copying

Reply:
ok, do you mind rename `hash` -> `info` (or `infoHash`)? I find the old name easily mixed up with the element type of the container, and the ones below.

Reply:
Certainly.

Reply:
I did it in the coding style commit.